.buildkite/pipelines/esql_grammar_sync.yml

```diff
@@ -1,13 +1,13 @@
 steps:
-  - command: .buildkite/scripts/steps/esql_grammar_sync.sh
-    label: Grammar Sync
-    timeout_in_minutes: 10
-    agents:
-      image: family/kibana-ubuntu-2004
-      imageProject: elastic-images-prod
-      provider: gcp
-      machineType: n2-standard-2
-      preemptible: true
+  # - command: .buildkite/scripts/steps/esql_grammar_sync.sh
```

Comment:
Why this change? Which is the difference with the commented part?

Reply:
This prevents the ANTLR sync